src/components/App.tsx

```diff
@@ -962,7 +962,7 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         zoom: this.state.zoom,
         remotePointerViewportCoords: pointerViewportCoords,
         remotePointerButton: cursorButton,
-        remoteSelectedElementIds: remoteSelectedElementIds,
```

Comment:
not a fan of the eslint rule because we'll end up with less readable cases like this

Reply:
I'll disagree with the "less readable" as you have to read less, and we were already using this rule in various places in the codebase.. this just makes it autofixable and consistent..

Reply:
Anyway, let's remove the `roomId` change and we can merge